### ISSUE TITLE
No required attribute when using OneOf

### ISSUE THREAD

**Issue by goganchic** [apiaryio/drafter#452]
apib file:
```
# GET /ab
- response 200 (application/json)
    - Attributes (X)

# Data Structures

## X
- One Of
    - Properties
        - foo: A (string, required)

    - Properties
        - bar: B (string, required)
```

generated json schema:
```
{
  "$schema": "http://json-schema.org/draft-04/schema#",
  "type": "object",
  "properties": {},
  "oneOf": [
    {
      "properties": {
        "foo": {
          "type": "string"
        }
      }
    },
    {
      "properties": {
        "bar": {
          "type": "string"
        }
      }
    }
  ]
}
```

I'm expecting to see something like:
```
{
  "$schema": "http://json-schema.org/draft-04/schema#",
  "type": "object",
  "properties": {},
  "oneOf": [
    {
      "properties": {
        "foo": {
          "type": "string"
        }
      },
      "required": [
        "foo"
      ]
    },
    {
      "properties": {
        "bar": {
          "type": "string"
        }
      },
      "required": [
        "bar"
      ]
    }
  ]
}
```

Is it a bug or I'm missing something?

**Comment by pksunkara** [comment]
Yes, this is a bug.

**Comment by kylef** [comment]
Fixed via #453
